torch/distributions/multinomial.py

```diff
@@ -103,6 +105,20 @@ def sample(self, sample_shape=torch.Size()):
         counts.scatter_add_(-1, samples, torch.ones_like(samples))
         return counts.type_as(self.probs)
 
+    def entropy(self):
+        n = torch.tensor(self.total_count)
+
+        cat_entropy = self._categorical.entropy()
+        log_factorial_n = torch.lgamma(n + 1) * torch.ones_like(cat_entropy)
```

Comment:
out of curiosity, why do you need the `* torch.ones_like(cate_entropy)`? Shouldn't that already happend in the broadcast below?

Reply:
Fixed.